site/src/scss/main.scss

```diff
@@ -1,6 +1,4 @@
-
-@import "variables";
-@import "~foundation-sites/scss/foundation";
+@import "init";
```

Comment:
@rbayliss I removed the `variables` and `foundation` imports because they were already imported in `init`.